### PR TITLE
Adding mssql as a test db

### PR DESCRIPTION
**Pull request by paoliniluis**

### CODE REVIEW

**Review by metamben — CHANGES_REQUESTED** [review]
- There are some copy&paste errors. 
- I hope we can get an arm64 image too.
- I couldn't build the image -- I was getting a login error:
  > 2023-03-03 20:31:39.55 Logon       Login failed for user 'sa'. Reason: An error occurred while evaluating the password. [CLIENT: 127.0.0.1]

File: .github/workflows/build-and-push.yml [hunk 1]
```diff
@@ -126,3 +126,24 @@ jobs:
           password: ${{ secrets.DOCKERHUB_TOKEN }}
       - name: Create oracle 21 image
         run: docker buildx build -f dbs/oracle/Dockerfile -t metabase/qa-databases:oracle-xe-21.3 --push --compress --no-cache .
+
+  build-mssql:
+    runs-on: ubuntu-22.04
+    timeout-minutes: 30
+    strategy:
+      matrix:
+        version: [2017, 2019, 2022]
```
[the remark this answers is not in the document]
Comment (paoliniluis) [reply .github/workflows/build-and-push.yml]: they have a latest tag which is 2019, we can have a latest tag as well but as it seems that the Dockerfile changes on every latest tag the build will fail eventually

**Comment by paoliniluis** [comment]
I think they only ship amd64 arch for this stuff https://hub.docker.com/_/microsoft-mssql-server

**Comment by metamben** [comment]
> I think they only ship amd64 arch for this stuff https://hub.docker.com/_/microsoft-mssql-server

Can you check this again? Unless I'm confusing things, I could find an arm64 image.

File: .github/workflows/build-and-push.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (metamben) [reply .github/workflows/build-and-push.yml]: Is there no 2022 version?

**Comment by paoliniluis** [comment]
just did a re-check, only amd64, no aarch64 or armv8 stuff

**Review by metamben — APPROVED** [review]

**Pull request merged**